spec/controllers/ops_controller/settings/schedules_spec.rb

```diff
@@ -59,12 +59,12 @@
 
       it "responds with a filtered cluster list" do
         json = JSON.parse(response.body)
-        expect(json["filtered_item_list"]).to eq([%w(clustertest__datacenter desc)])
+        expect(json["filtered_item_list"]).to eq([['clustertest__datacenter', 'clustertest in datacenter']])
```

Comment:
Is the filtered_item_list guaranteed to return the array in this order?  If order isn't guaranteed, we should use match_array here.